src/utils/components/ActionDropdownItem/ActionDropdownItem.tsx

```diff
@@ -20,6 +20,7 @@ const ActionDropdownItem: FC<ActionDropdownItemProps> = ({ action, setIsOpen })
 
   const actionAllowed = accessReview || action?.accessReview === undefined;
   const isCloneDisabled = !actionAllowed && action?.id === 'vm-action-clone';
+  const displayDisabledTooltip = action?.disabledTooltip && action?.disabled;
```

Comment:
`isCloneDisabled`  seems a special case that should merged into generic flow. If it's out-of-scope then we could try to merge both "disable" paths as early as possible. Ideally we should have one `isDisabled` flag and one tooltip.

Reply:
Great catch! Fixed!